support-frontend/app/services/stepfunctions/SupportWorkersClient.scala

```diff
@@ -26,27 +26,37 @@ import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
 object CreateSupportWorkersRequest {
+
   import codecs.CirceDecoders._
+
   implicit val codec: Codec[CreateSupportWorkersRequest] = deriveCodec
 }
+
 case class CreateSupportWorkersRequest(
-    firstName: String,
-    lastName: String,
-    billingAddress: Address,
-    deliveryAddress: Option[Address],
-    product: ProductType,
-    firstDeliveryDate: Option[LocalDate],
-    paymentFields: PaymentFields,
-    promoCode: Option[PromoCode],
-    ophanIds: OphanIds,
-    referrerAcquisitionData: ReferrerAcquisitionData,
-    supportAbTests: Set[AbTest],
-    email: String,
-    telephoneNumber: Option[String]
+  title: Option[Title],
```

Comment:
Curious - does this make https://trello.com/c/qmya13nc/2404-checkout-forms-do-not-store-title-field any easier? Or even resolve it?

Reply:
It resolves it, I thought I might as well do it while I'm here!